emData/dtclinklayerdisk.dat

```diff
@@ -0,0 +1,25 @@
+2S_1 4 5 -1
```

Comment:
This file is already in the tarball downloaded by `download.sh`. Can we just use that, and not add a separate copy of it to the repo?

Reply:
Sorry, should have removed it.